.github/workflows/build.yml

```
@@ -0,0 +1,34 @@
+name: Build Fobnail binaries
+
+on:
+  push:
+
+env:
+  FOBNAIL_SDK_VERSION: v0.2.3
```

@macpijan Updating to v0.2.4 (after https://github.com/fobnail/fobnail-sdk/pull/8) should fix problem with building on nRF.
```suggestion
  FOBNAIL_SDK_VERSION: v0.2.4
```

---

please change and test

---

Done